### ISSUE TITLE
Add one hot encoding (OHE) utility to ML workbench

### ISSUE THREAD

**Issue by kxk302** [bgruening/galaxytools#1093]
When doing multi-class classification, the labels file need to be presented in one-hot encoding to a neural network, so categorical cross entropy can be calculated during training. Currently, no ML workbench tool can convert the format to OHE. 

keras.utility.to_categorial offers this functionality. Need to wrap it so the tool is available to Galaxy.
